swift/Samples/Tutorial/Frameworks/Tutorial5Complete/Tests/TodoWorkflowTests.swift

```diff
@@ -77,21 +78,21 @@ class TodoWorkflowTests: XCTestCase {
                     // We always expect the TodoListWorkflow
                     ExpectedWorkflow(
                         type: TodoListWorkflow.self,
-                        rendering: BackStackScreen.Item(
+                        rendering: BackStackScreen<AnyScreen>.Item(
```

Comment:
As we talked about in our 1:1, I think we can snip these `<AnyScreen>` and let the compiler infer the generic type.